commit f8cef9e6ca9886c373c0975e53b7d4d17f0b256e
Author: Matthias Reichl <hias@horus.com>
Date:   Fri Feb 2 20:30:42 2018 +0100

    ASoC: allo-piano-dac-plus: fix S24_LE format
    
    Remove set_bclk_ratio call so 24-bit data is transmitted in
    24 bclk cycles.
    
    Signed-off-by: Matthias Reichl <hias@horus.com>

diff --git a/sound/soc/bcm/allo-piano-dac-plus.c b/sound/soc/bcm/allo-piano-dac-plus.c
index 2749b4fd8ac3..bede9bdec3c7 100644
--- a/sound/soc/bcm/allo-piano-dac-plus.c
+++ b/sound/soc/bcm/allo-piano-dac-plus.c
@@ -794,9 +794,6 @@ static int snd_allo_piano_dac_hw_params(
 		struct snd_pcm_hw_params *params)
 {
 	struct snd_soc_pcm_runtime *rtd = substream->private_data;
-	struct snd_soc_dai *cpu_dai = rtd->cpu_dai;
-	unsigned int sample_bits =
-		snd_pcm_format_physical_width(params_format(params));
 	unsigned int rate = params_rate(params);
 	struct snd_soc_card *card = rtd->card;
 	struct glb_pool *glb_ptr = card->drvdata;
@@ -838,8 +835,6 @@ static int snd_allo_piano_dac_hw_params(
 	if (ret < 0)
 		return ret;
 
-	ret = snd_soc_dai_set_bclk_ratio(cpu_dai, sample_bits * 2);
-
 	return ret;
 }
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1200_f8cef9e6ca9886c373c0975e53b7d4d17f0b256e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1200_f8cef9e6ca9886c373c0975e53b7d4d17f0b256e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
